src/main.js

```diff
@@ -41,12 +41,14 @@ import { formatMoney } from '@/vue/filters/formatMoney'
 import { formatNumber } from '@/vue/filters/formatNumber'
 import { formatOrderNumber } from '@/vue/filters/formatOrderNumber'
 import { formatInteger } from '@/vue/filters/formatInteger'
+import { formatPercent } from '@/vue/filters/formatPercent'
 import { globalize } from '@/vue/filters/globalize'
 
+import { globalizeFeeType } from '@/vue/filters/globalizeFeeType'
```

Comment:
I'd make this filter the formatter also. Call it `formatFeeType` and move all the logic to `i18nOptions.format` method

Reply:
Resolved, updated

Reply:
Resolved, updated